opentok/opentok-meet
# [OPENTOK-44710] Adding missing information back to stats

#### What is this PR doing?
Fix a bug which was not adding streamId/cascading information to the stats.

#### How should this be manually tested?
- Run the app, join to the same session form 2 different tabs.
- Click on subscriber stats.
- Check _streamId_ is now present.
- Check origin/edge server names are also present.

- Please also check this info is missing in meet.tokbox.com

#### What are the relevant tickets?
OPENTOK-44710
